app/firebase/functions/src/users/users.helper.ts

```diff
@@ -108,14 +112,43 @@ export class UsersHelper {
     return platformIds;
   }
 
-  /** return a list of platformIds which have at least one account that hasn't yet fetched from platform */
-  static platformsWithoutFetch(user: AppUser): PUBLISHABLE_PLATFORM[] {
```

Comment:
this was to make sure the initial fetch was triggered for newly added platforms.